### PR TITLE
[webui][api] Move options from QueryBuilder up to ForSource

### PR DESCRIPTION
**Pull request by ChrisBr**
because it is also required for accepted requests.

### CODE REVIEW

**Comment by DavidKang** [comment]
@ChrisBr, please rebase with master :smile_cat:

File: src/api/app/models/bs_request_action/differ/for_source.rb [hunk 1]
```diff
@@ -35,6 +34,8 @@ def query
         query = {}
         query[:view] = :xml if options[:view].to_s == 'xml'
         query[:withissues] = 1 if options[:withissues].present?
+        query[:filelimit] = options[:filelimit] ? options[:filelimit].to_i : 10_000
```
Comment (bgeuken) [inline src/api/app/models/bs_request_action/differ/for_source.rb]: How about ` query[:filelimit] = (options[:filelimit] ||10_000).to_i `?

Comment (bgeuken) [reply src/api/app/models/bs_request_action/differ/for_source.rb]: Ah, you just moved the code over

File: src/api/spec/models/bs_request_action/differ/for_source_spec.rb [hunk 2]
```diff
@@ -54,8 +55,8 @@
           rev:        2,
           oproject:   target_project.name,
           opackage:   target_package.name,
-          filelimit:  42,
-          tarlimit:   43,
+          filelimit:  42, # options
+          tarlimit:   43, # options
```
Comment (bgeuken) [inline src/api/spec/models/bs_request_action/differ/for_source_spec.rb]: I doubt that anyone in the future will know what these comments mean :thinking:

**Comment by ChrisBr** [comment]
> do not specify a fallback value in api, this gets usually defined in the backend,
if needed.

I just moved the code!

**Comment by bgeuken** [comment]
@adrianschroeter Do you have any concerns merging this? Otherwise I would do so.

**Comment by ChrisBr** [comment]
@adrianschroeter @bgeuken Btw: removing the defaults from the frontend would introduce regressions! So you think this is a problem I would prefer that we create a card and fix it in one of the next sprints.

**Comment by adrianschroeter** [comment]
no concern, I have not noticed that it just got moved

**Pull request merged**